src/client/app/components/home/home.component.ts

```diff
@@ -27,12 +27,16 @@ export class HomeComponent {
     return false;
   }
 
-  readAbout() {
+  readAbout(event: Event) {
     this.routerext.navigate(['/about'], {
       transition: {
         duration: 1000,
         name: 'slideTop',
       }
     });
+
+    if (event && 'preventDefault' in event) {
```

Comment:
It will cause the link to not show up blue, but that's just a styling mark at that point. You could add a `a` style in the home.component.css if you want it to show up as such.